conda_smithy/ci_register.py

```diff
@@ -220,9 +226,6 @@ def add_project_to_travis(user, project):
             print(' * Registered on travis-ci')
         else:
             raise RuntimeError('Unable to register on travis-ci, response from hooks was negative')
-        url = '{}/users/sync'.format(endpoint)
-        response = requests.post(url, headers=headers)
-        response.raise_for_status()
```

Comment:
I'm a little unclear on what this was doing before. However it seems to have been [intentionally]( https://github.com/conda-forge/conda-smithy/commit/60d2c3d7ad1792cdd232be8c1fa532b038ccfc3d#diff-8666415dcccfaf789819c990341084c8R120 ) added. So it would be good if we could understand why it was here before we drop it.